### ISSUE TITLE
glob: windows support

### ISSUE THREAD

**Issue by sils1297** [coala-analyzer/coala#496]
According to @Makman2 this is probably the only reason for us not yet supporting windows. Any investigation results until now?

**Comment by AbdealiJK** [comment]
according to the Python docs, glob should be supported on Windows. @Makman2 -  What exactly is the trouble ?

**Comment by sils1297** [comment]
We have our own globbing implementation because we support recursive
globbing.
Am 04.05.2015 04:24 schrieb "AbdealiJK" <notifications@github.com>:

**Comment by smfn3321** [comment]
Ok I'm looking at this right now. 

This is connected to `os.path.join()` only identifying a drive letter as such if it is followed by backslashes. 
Examples:
```
`>>> os.path.join("foo", "bar")
'foo\\bar'
>>> os.path.join("foo", "c:")
'foo\\c:'
>>> os.path.join("foo", "bar\\")
'foo\\bar\\'
>>> os.path.join("foo", "c:\\")
'c:\\'
```
so watch out if you use something similar.

**Comment by smfn3321** [comment]
addressed in  #501

I will leave this open until #501 is accepted

**Comment by sils1297** [comment]
pulled to #475
